src/opnsense/scripts/interfaces/traffic_top.py

```diff
@@ -94,15 +95,32 @@ async def request_ittr(self, addresses):
         dnsResolver.timeout = 2
         tasks = []
         for address in addresses:
-            tasks.append(dnsResolver.resolve_address(address))
+            try:
+                if ":" in address:
+                    ipv6_obj = ipaddress.IPv6Address(address)
+                    reverse_ip = '.'.join(reversed(ipv6_obj.exploded.replace(":", ""))) + ".ip6.arpa"
+                else:
+                    reverse_ip = '.'.join(reversed(address.split("."))) + ".in-addr.arpa"
+
+                tasks.append(dnsResolver.resolve(reverse_ip, "PTR"))
+            except Exception as e:
```

Comment:
better to catch explicit exception types, unexpected errors are better be thrown so you know what to look for later

Reply:
I don't really write in python, imo any exception here is acceptable to skip that address in the lookup process.